sites/NOAA_GFDL/mdtf_gfdl.csh

```diff
@@ -27,18 +27,19 @@ set staticfile
 set fremodule
 set script_path
 
-## set paths
+## set paths to site installation
 set REPO_DIR="/home/Oar.Gfdl.Mdteam/DET/analysis/mdtf/MDTF-diagnostics"
```

Comment:
the paths need to be updated to the new central install location : 
`/home/oar.gfdl.mdtf/mdtf/MDTF-diagnostics` 
` /home/oar.gfdl.mdtf/mdtf/inputdata`

Reply:
Thanks for the reminder! I had run `ls /home/oar.gfdl.*` but forgot the `-d` and missed the listing in the output -- figured the account wasn't set up yet.

Reply:
Fixed in 0bcae3c94530d13df7ff41972b38f26937af9242